BrowserKit/Sources/MenuKit/MenuSection.swift

```diff
@@ -7,11 +7,18 @@ import Foundation
 public struct MenuSection: Equatable {
     public let isHorizontalTabsSection: Bool
     public let isExpanded: Bool?
+    public let isHomepage: Bool
     public let options: [MenuElement]
 
-    public init(isHorizontalTabsSection: Bool = false, isExpanded: Bool? = false, options: [MenuElement]) {
+    public init(
+        isHorizontalTabsSection: Bool = false,
```

Comment:
in previous PR only Homepage had horizontal tab section, does this change means we will have the section for websites also?